components/content/inspira/ui/text-reveal-card/TextRevealStars.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+  <div class="absolute inset-0">
+    <Motion
+      v-for="i in 130"
```

Comment:
I think we should add a prop like `count` to handle the number of stars dynamic, giving user more freedom.